packages/governance/src/binaryVoteCounter.js

```diff
@@ -113,6 +119,13 @@ const makeBinaryVoteCounter = (questionSpec, threshold, instance) => {
 
     if (!makeQuorumCounter(threshold).check(stats)) {
       outcomePromise.reject('No quorum');
+      /** @type {OutcomeRecord} */
+      const voteOutcome = {
+        question: details.questionHandle,
+        outcome: 'fail',
+        reason: 'No quorum',
```

Comment:
you mentioned that "quorum" isn't the right term for this. Now may be a good time to improve the message. We can at least provide the `threshold` value for diagnostics.

e..g. `Too few voters for threshold ${threshold}`

Reply:
I'm resisting scope creep. I filed an issue (#6209) to fix the terminology from "quorum" to "threshold".   The term "quorum" appears in too many places to want to fix it at the last minute.